apps/web/package.json

```diff
@@ -76,17 +76,20 @@
     "focus-visible": "^5.2.0",
     "framer-motion": "^10.12.16",
     "hono": "^3.2.5",
+    "jotai": "^2.2.2",
```

Comment:
where did this come from?

Reply:
yeah will remove this it was initial to make modals to open when creating workspace and inviting members but I'm not doing that now